src/exclusive-or.rst

```diff
@@ -321,52 +321,52 @@ some sequences that we know will occur very commonly, for example (the
 -  :math:`\verb*| and |` (no variants; only occurs in the middle of a sentence)
 -  :math:`\verb*| a |` and variants
 
-If we know more about the plaintext, we can make even better guesses.
-For example, if it's HTTP serving HTML, we would expect to see things
-like ``Content-Type``, ``<a>``, and so on.
-
-That only tells us which plaintext sequences are likely, giving us
-likely guesses. How do we tell if any of those guesses are correct? If
-our guess is correct, we know all the other plaintexts at that position
-as well, using the technique described earlier. We could simply look at
+We can make better guesses if more information is known about the plaintext.
+For example, if HTTP is serving HTML we expect to see
```

Comment:
maybe something like "HTML is served over HTTP"

Reply:
Sounds good, go for it!